sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -58,6 +60,8 @@
     protected String octaneUserValue;
     protected final String urlDomain;
     protected Authentication lastUsedAuthentication;
+    private final Map<OctaneHttpRequest, OctaneHttpResponse> cachedRequestToResponse = new HashMap<>();
```

Comment:
Is it not concerning that we're saving every response in a map for ever and ever?